ref/feature/samlWeb/examples.adoc

```diff
@@ -0,0 +1,33 @@
+**Examples**
+
+The `samlWebSso20` element is configured by default in Liberty.
+
+----
+<samlWebSso20 id="defaultSP">
+
+</samlWebSso20>
+----
+
+The following default values are assumed in this default configuration,
+
+•	AssertionConsumerService URL:
+
+`https://<hostname>:<sslport> /ibm/saml20/defaultSP/acs`
+
+•	Service Provider (SP) metadata URL:
+
+`https://<hostname>:<sslport> /ibm/saml20/defaultSP/samlmetadata`
+
+This URL is used to download the metadata for the service provider (SP) and provide the URL to the SAML identity provider to establish federation between this SP and Identity Provider (IdP).
+
+The IdP metadata file must be copied to the `resources/security` directory on the server, and named `idpMetadata.xml`.
+The issuer name for SAML assertion is used as the security realm, and NameID is used as the principal to create an authenticated subject from the SAML assertion.
+
+The SAML `AuthnRequest` is signed with a private key in the default keystore of the server if the attribute `KeyStoreRef` is not specified. If the `keyAlias` is not configured, then samlsp is the default key alias.
+If the `keyAlias` is not configured, and the keystore only contains one private key, the private key is used in the signature.
+
+If you create a new service provider instance, and the defaultSP is no longer required, then the defaultSP instance must  be explicitly disabled, by adding the following to the `server.xml` file:
```

Comment:
- what is defaultSP?
- **change** If you create a new service provider instance, and the defaultSP is no longer required, then the defaultSP instance must  be explicitly disabled, by adding the following to the `server.xml` file:
**to**
If you create a new service provider instance and the `defaultSP` instance is no longer required, then you must explicitly disable the `defaultSP` instance. You can disable the defaultSP instance by adding the following code to the `server.xml` file: